reviews/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db  import models
+
+from core.models    import TimeStampedModel
+from products.models import Product
+from users.models   import User
+
+class Review(TimeStampedModel):
+    product       = models.ForeignKey(Product, on_delete=models.CASCADE)
+    user          = models.ForeignKey(User, on_delete=models.CASCADE)
+    parent        = models.ForeignKey('self', on_delete=models.CASCADE)
```

Comment:
parent 보다는 `parent_review`로 이름을 정하는게 더 명확할 것 같습니다.

```suggestion
    parent_review = models.ForeignKey('self', on_delete=models.CASCADE)
```